### DISCUSSION TITLE
Wider support in container scanners

### DISCUSSION THREAD

**Discussion by jbergstroem** [wolfi-dev/os#1736] [Ideas]
Hey! Stumbled over wolfi / chainguard recently. The quality of the maintained images is something else!

I'd love to see stronger support in container scanners such as:
- [Amazon Inspector ECR Scanning](https://docs.aws.amazon.com/inspector/latest/user/enable-disable-scanning-ecr.html), 
- [AWS Elastic Container Registry (ECR](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html))
- [GCP Container Registry (GCR](https://cloud.google.com/container-analysis/docs/vulnerability-scanning))
- [Azure Defender for Container Registries - Container Scanning](https://docs.microsoft.com/en-us/azure/security-center/defender-for-container-registries-introduction)

Most of these seems to use Clair or Trivy under the hood to some extent. I [opened an issue in the clair repo](https://github.com/quay/claircore/issues/923) about growing support for Wolfi containers which shouldn't be too hard seeing how its basically Alpine Linux under the hood.

I understand the benefits of the chainguard enforce product/ecosystem. I think it could be complementary to having green lights in these scanners as well!

**Comment by dlorenc** [comment]
Hey!

Thanks for opening this! We're working wherever we can to add support in scanners. The easiest way is actually for end users like you to file feature requests in those products wherever possible. We just got support added into grype and trivy, and are working on getting support added in Twistlock and Snyk now.
